### DISCUSSION TITLE
difference between merging and integration of scATAC datasets

### DISCUSSION THREAD

**Discussion by pengxin2019** [stuart-lab/signac#581] [Q&A] [answered]
Hi Tim,
I am kind of confused between these  two pipelines  even after I read through it.
1. scATAC
scATAC integration: https://satijalab.org/signac/articles/integration.html
scATAC merging: https://satijalab.org/signac/articles/integration.html

What is the criteria for us to choose the correct pipeline? Is it correct that we should use merging pipeline if the samples are PBMCs and should use integration if the samples are tissues?

2. In terms of the QC of scATAC listed here: https://satijalab.org/signac/articles/pbmc_vignette.html
Should we merge/integrate biologically replicates of scATAC and use the merged/integrated one as the input of QC? Or actually you suggest using each of biological replicate as the input of QC step?

3. integration of scATAC and scRNA
Also, what is the difference between these two pipeline? 
https://satijalab.org/seurat/articles/atacseq_integration_vignette.html
https://satijalab.org/signac/articles/pbmc_vignette.html

It seems like both of them are talking about  integration of scRNA and scATAC, but the requirements of the input files are different? How should we choose these two pipeline?

Thanks
Best

**Comment by timoast** [comment, marked as answer]
> What is the criteria for us to choose the correct pipeline?

The integration methods are designed to remove confounding dataset-specific differences. The question of whether you should apply these methods depends on whether there are confounding dataset-specific differences that are an issue for your analysis. A simple way to assess this is generating a UMAP dimension reduction and observing whether cells group by both cell type and dataset of origin.

> Is it correct that we should use merging pipeline if the samples are PBMCs and should use integration if the samples are tissues?

No, see above answer

I would recommend performing QC and removing low-quality cells before integrating or merging datasets together.

**Reply by pengxin2019** [reply]
Hi Tim, thanks for getting back to me. Since my samples are two replicates of PBMC and I tried merged directly and found that in this plot "DimPlot(combined, group.by = 'dataset', pt.size = 0.1)" of merging pipeline (https://satijalab.org/signac/articles/merging.html) are not overlapping well. (attached below) The faction of the non-overlapping is unexpected. Then I did the QC and removing of low quality cells. 

![Screen Shot 2021-04-23 at 7 12 01 AM](https://user-images.githubusercontent.com/54915282/115869264-425a3f80-a403-11eb-89ee-82d6a4c62934.png)

Do you recommend me performing QC and removing low-quality cells first (https://satijalab.org/signac/articles/pbmc_vignette.html) then perform scATAC data integratio(https://satijalab.org/signac/articles/integration.html)   (or I should keep using merging pipeline)? 
Thanks

**Reply by pengxin2019** [reply]
by the way, the dims = 2:50 for the plot above

**Reply by pengxin2019** [reply]
also, i just realized that scATAC data integration pipeline is using some rds file as input which I do not have. I do have the files used in merging pipeline. So, I would not be able to use scATAC data integration pipeline to remove the confounding dataset-specific differences anyway?

**Reply by timoast** [reply]
The integration vignette begins with a pre-processed dataset for convenience. As explained in the vignette, the full code used to generate that object is described in a separate vignette. To answer your question of whether you need to used the integration methods, please read the first answer to your question that I posted above

**Reply by pengxin2019** [reply]
Hi Tim,
Thanks for getting back to me.
I have two independent biological replicates and performed the QC using the pipeline (https://satijalab.org/signac/articles/pbmc_vignette.html).

Their TSS enrichment pattern looks very different using this pipeline:
![Screen Shot 2021-04-28 at 3 30 03 PM](https://user-images.githubusercontent.com/54915282/116468558-9c5d6980-a836-11eb-9ef8-5ddc8048800d.png)

As you can see, pattern of low TSS score in the left replicate looks weird but there is now low TSS plot in the right replicate which means all of the TSS score in the right replicate are high.

But in the Cell Ranger ATAC, the TSS pattern of these two replicates looks very similar:
![Screen Shot 2021-04-28 at 3 33 54 PM](https://user-images.githubusercontent.com/54915282/116468968-26a5cd80-a837-11eb-9c86-f59cf116ac42.png)

Did you have this problem when you developed this pipeline? I do not know why this is happening.

Thanks

**Reply by timoast** [reply]
There is no problem here, you likely have very few cells in the "low TSS enrichment" category that you've defined. This is getting off topic
